### ISSUE TITLE
/ghost/api/v0.1/schedules/posts/... schedule request HTTP status 404

### ISSUE THREAD

**Issue by kolbma** [TryGhost/Ghost#10039]
### Issue Summary

The scheduling of posts doesn't work.
The only thing I see in logs is that there is a GET request to /ghost/api/v0.1/schedules/posts/5bc89644f2572d7e9bdb13ea?client_id=ghost-scheduler&client_secret=xxxxxxxxxxx which has HTTP status 404.
Always 404 for these requests.

### To Reproduce

Create a post and activate schedule for later.
Scheduled posts are never published.

### Technical details:

* Ghost Version: 2.2.3
* Node Version: v8.12.0
* Browser/OS: Firefox Windows
* Database: SQLite3

**Comment by kirrg001** [comment]
Please use our [forum](https://forum.ghost.org) for help.

e.g. [see this thread](https://forum.ghost.org/t/scheduled-posts-dont-publish/1179/33?u=kate)

**Comment by kolbma** [comment]
@kirrg001 Why you are closing immediately? What's up with fixing your scheduler to follow Redirects which is HTTP standard?!

I don't like to put the Reverse Proxy up to port 80. And I don't want to listen Ghost on port 443. So I need to configure Ghost config url to http:// and set proxy forward protocol to https so that the links in Ghost work.

So why your scheduler is not using the same procedure as the rest of the app to use the protocol from the proxy forward?!

**Comment by kolbma** [comment]
Also if I think something more about it, it would be better to fix the server setup. So that you can really specify the correct external url in the config, but say something in the server config to not use ssl setup.

**Comment by kolbma** [comment]
Ok. Sorry. Partly my fault. I've got SSL handshake failures and logs about missing cert and key and info about LetsEncrypt during setup when using https in config.url.

But after looking through your code, there is no way to start express with ssl by config.
So seems to be only a setup and first run problem.

I've switched config.url to https and it uses correct urls over the place.

Because it works for me now, it's your job if you'd like a hacked or working product.
